hackmd-day3.md

```diff
@@ -0,0 +1,260 @@
+      - Rebasing is advanced and it modifies history and it is OK to not use it.
+      - It can help creating more linear histories.
+      - Ok, got it. Thanks!
+
+- how to add a colaborator in a project on github? Is it under setting/action permission/add runner?
+  - settings -> "manage access"
+
+
+### Day 3 feedback
+
+* Pro: The breakoutroom worked much better today. Examples are very clear and gave us good experience of working collaboratively.
+* Cons: The only thing is to keep the  text in one place to not confuse people with what the text is. Try to be consistent in time structure for class (when breaks are).
+
+* For the breakout rooms we managed to do very little, and that little was not even always correct. Next time that will have to be better prepared and at least tested, to make sure the instructions are up to date.
+
+* The examples with central->distributed were very clear, to improve: some were missunderstanding "examples" as lines that should have been executed. The remote upstream namely would benefit from different backround for "do this", and "read this". But I liked a lot.
+
+* The first part of the main session was good, although really fast and I am not sure those not familiadayr were all able understand.
+* The second part was really confusing and difficult to follow (Richard even intervened to ask what that was about) especially when the trainer went back & forth in the teaching material or did not share his terminal while typing.
+
+* A thorough recap next Tuesday at the start of the session about what we did today will be highly appreciated.
+
+- The main reason I registered for this workshop was to know more about "Collaborative distributed version control" and learn the best practices through simple exercises. I get out of here really frustrated.
+  - [name=Radovan] sorry to read this. I would love to hear what I/we can do to avoid this in future. Was it the presentation? The material? The exercises/examples? The timing? Speed?
+
+- Feedback:
+    - Pros:
+        - Very well designed exercises
+        - Clear and easy to follow explanations
+    - Cons:
+        - Time allocated to exercises a bit short
+    - Overall: a big big up for the effort, the material and the energy put in this. Thank you!
+
+* Maybe next time we could show how to do it firstly, then we redo it in our group. I think it is easy for following. This morning, I did not get most of lessons.
+
+- In our breakout rooms one person litterally monopolized all attention of the helper, systematically. That slowed down everything and significantly affected our sessions. It may be better, perhaps, to put absolute beginners together into a separate group and adapt the the training for them.
```

Comment:
```suggestion
- In our breakout rooms one person literally monopolized all attention of the helper, systematically. That slowed down everything and significantly affected our sessions. It may be better, perhaps, to put absolute beginners together into a separate group and adapt the the training for them.
  - We will try to make things are more balanced in the future.  It's quite hard to learn to say no to someone, so this is always a concern.
```